.ci/install.sh

```diff
@@ -18,3 +18,5 @@ elif [[ "${BUILD_MINGW}" == ON ]]; then
 fi
 
 pip install --user --upgrade cpp-coveralls neovim
+pip-3.3 install --user --upgrade neovim
```

Comment:
That's what I suspected. So, next question is, is there a way to set up python withou sudo and without having to add a permutation to the build matrix...